base/sim/FairGenerator.h

```diff
@@ -52,6 +52,9 @@ class FairGenerator : public TNamed
     /**Initialize the generator if needed */
     virtual Bool_t Init() { return kTRUE; }
 
+    /**Finalize the generator if needed */
+    virtual Bool_t Finish() { return kTRUE; }
```

Comment:
The return value is not used at all in our code.

So let's just return `void`, please.

In exceptional cases, people can still throw exceptions.